CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ BUG FIXES:
 * `r/tfe_workspace_settings`: Allow multiple unknown values in `remote_state_consumer_ids`, by @brandonc [#1569](https://github.com/hashicorp/terraform-provider-tfe/pull/1569)
 
 FEATURES:
+
+* `r/tfe_project`: Add `auto_destroy_activity_duration` field to the project resource, which automatically propagates auto-destroy settings to workspaces [1550](https://github.com/hashicorp/terraform-provider-tfe/pull/1550)
+* `d/tfe_project`: Add `auto_destroy_activity_duration` field to the project datasource [1550](https://github.com/hashicorp/terraform-provider-tfe/pull/1550)
```

Comment:
these should go in the unreleased, I think they may have got pulled into v0.63.0 when merging main

Reply:
I fixed it, it was indeed merged into main.